tests/functional/rbac/test_reassignment.py

```diff
@@ -149,6 +154,137 @@ def grant_role(self, client: ADCMClient, user: User, role: RbacRoles, *objects:
                 name=f'{user.username} is {role.value}', role=client.role(name=role.value), objects=objects, user=[user]
             )
 
+    def test_add_remove_user_from_group_and_policy(self, clients, is_denied_to_user, prepare_objects, user):
+        """Test that policies are applied after user add/remove to the policy"""
+
+        admin_cluster, admin_service, admin_component, *_ = prepare_objects
+
+        is_denied_to_user(admin_cluster, BusinessRoles.EditClusterConfigurations)
+        is_denied_to_user(admin_service, BusinessRoles.EditServiceConfigurations)
+        is_denied_to_user(admin_component, BusinessRoles.EditComponentConfigurations)
+        with allure.step('Create user group'):
+            test_group = clients.admin.group_create("Test_group", user=[{"id": user.id}])
+        with allure.step('Create a test policy with user group'):
+            test_policy = clients.admin.policy_create(
+                name='Test_policy',
+                role=clients.admin.role(name=RbacRoles.ClusterAdministrator.value),
+                user=[],
+                group=[test_group],
+                objects=[admin_cluster],
+            )
+        user_cluster, user_service, user_component = as_user_objects(
+            clients.user, admin_cluster, admin_service, admin_component
+        )
+        is_allowed(user_cluster, BusinessRoles.EditClusterConfigurations)
+        is_allowed(user_service, BusinessRoles.EditServiceConfigurations)
+        is_allowed(user_component, BusinessRoles.EditComponentConfigurations)
+        with allure.step('Change test group: delete user'):
```

Comment:
I suggest not using "test" here and in other places. I thinks it's the same as just "group"

Reply:
why?

Reply:
IMHO "test" doesn't give any more info for the report reader. It does when we have more than one group, naming one "test" and another "berebek" or something :D

Reply:
IMHO it is a nitpick, but okay, I'll change